Makefile

```diff
@@ -17,3 +17,28 @@ install:
 start: prep
 	npm start
 
+
+
+all: devel-install build devel-uninstall
+
+# this requires a built source tree and avoids having to install anything system-wide
+devel-install:
+	mkdir -p ~/.local/share/cockpit
+	ln  -sfn $(shell pwd)/cockpit/public $(INSTALL_DIR)
+	mkdir -p pkg/lib
+	git clone --depth 1 --branch main https://github.com/cockpit-project/cockpit.git temp_cockpit_repo
+	cp -r temp_cockpit_repo/pkg/lib/* pkg/lib/
+	rm -rf temp_cockpit_repo
+
+build:
```

Comment:
@schuellerf you had some suggestions about the makefile, right?
Do you mind having a look and adding your comments here please
🙏

Reply:
My "comment" is more a PR, I thought that's easier to get in line like this
https://github.com/mgold1234/image-builder-frontend/pull/12/commits

https://github.com/mgold1234/image-builder-frontend/pull/12/commits/496cd6607006afed0ae8bbad9879e530c16d3a40

Reply:
Okay, thanks I will have a look. I might end up cherry-picking that commit.

I just need to refer to the cockpit boilerplate [here](https://github.com/cockpit-project/starter-kit) and see what directories we will actually need. I don't think we need `pkg/lib` any more.

Reply:
Also I feel like github doesn't like the umlaut in your last name. Had to remove it to recognise you as a co-author 😅